devConfig.yaml

```diff
@@ -49,6 +49,11 @@ mysql_password: "root"
 # mysql_key_value_table_name:
 
 
+# (OPTIONAL | Default: "session_access_token_signing_keys") string value. Specify the name of the table that will store 
```

Comment:
Oh we don't need these config values for new tables. They were only for older tables.